crates/biome_service/src/file_handlers/css.rs

```diff
@@ -257,3 +257,9 @@ fn format_on_type(
     let printed = biome_css_formatter::format_sub_tree(options, &root_node)?;
     Ok(printed)
 }
+
+fn organize_imports(parse: AnyParse) -> Result<OrganizeImportsResult, WorkspaceError> {
```

Comment:
@ematipico 
What do you think if it's a safe change?
I've copied it from a `json.rs` file.

Reply:
Yes that's safe :)